src/Components/Aspire.Keycloak.Authentication/AspireKeycloakExtensions.cs

```diff
@@ -41,7 +45,11 @@ public static AuthenticationBuilder AddKeycloakJwtBearer(this AuthenticationBuil
     /// For example, if <paramref name="serviceName"/> is "keycloak" and <paramref name="realm"/> is "myrealm", the authority URL will be "https+http://keycloak/realms/myrealm".
     /// </remarks>
     public static AuthenticationBuilder AddKeycloakJwtBearer(this AuthenticationBuilder builder, string serviceName, string realm, string authenticationScheme)
-        => builder.AddKeycloakJwtBearer(serviceName, realm, authenticationScheme, null);
+    {
+        ArgumentNullException.ThrowIfNull(builder);
+
```

Comment:
```suggestion
        ArgumentException.ThrowIfNullOrEmpty(serviceName);
        ArgumentException.ThrowIfNullOrEmpty(realm);
        ArgumentException.ThrowIfNullOrEmpty(authenticationScheme);
```

Reply:
There is no need to check here, since the check occurs in the called method further
![image](https://github.com/user-attachments/assets/6bae3868-74c7-4f22-a7f6-4de19177cafa)
![image](https://github.com/user-attachments/assets/d0c958ad-ad3d-477e-a6bd-9e9d5e92d112)

Reply:
I khow but i perfer to check these first of methods.